zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -514,6 +514,22 @@ private boolean isCronUpdated(Map<String, Object> configA,
 
     return cronUpdated;
   }
+  private String normalizeNoteName(String path){
+    path = path.trim();
+    path = path.replace("\\", "/");
+    while (path.indexOf("///") >= 0) {
+      path = path.replaceAll("///", "/");
+    }
+    path = path.replaceAll("//", "/");
+    if (path.length() == 0) {
+      path = "/";
+    } else if (path.charAt(0) != '/'){
+      path = "/" + path;
```

Comment:
I think the method name is odd, This method is for the path not the notename. How about the method name is 'normalizeNotePath'?